ovh/types_cloud_project_database.go

```diff
@@ -129,11 +132,29 @@ type CloudProjectDatabaseCreateOpts struct {
 	Description  string                           `json:"description,omitempty"`
 	NetworkId    string                           `json:"networkId,omitempty"`
 	NodesPattern CloudProjectDatabaseNodesPattern `json:"nodesPattern,omitempty"`
+	Disk         CloudProjectDatabaseDisk         `json:"disk,omitempty"`
 	Plan         string                           `json:"plan"`
 	SubnetId     string                           `json:"subnetId,omitempty"`
 	Version      string                           `json:"version"`
 }
 
+type CloudProjectDatabaseDisk struct {
+	Type string `json:"type,omitempty"`
+	Size *int   `json:"size,omitempty"`
```

Comment:
really need pointer ? without it we can remove `GetNilIntPointer()` 
`d.Get("disk_size")`will give 0 when not set and that will not be push thanks to `omitempty`